tests/unit/s2n_client_supported_versions_extension_test.c

```diff
@@ -396,6 +392,62 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_connection_free(conn));
     }
 
+    /**
+     *= https://tools.ietf.org/rfc/rfc8446#appendix-D.2
+     *= type=test
+     *# A TLS server can also receive a ClientHello indicating a version number smaller than its highest supported
+     *# version. If the "supported_versions" extension is present, the server MUST negotiate using that extension as
+     *#  described in Section 4.2.1.
+     */
+    if (s2n_is_tls13_fully_supported()) {
+        {
```

Comment:
```suggestion

```
I would just use the if statement curly braces to separate out this test. Makes the test cleaner.